packages/client/src/__tests__/sigstore.test.ts

```diff
@@ -318,6 +319,17 @@ describe('#verify', () => {
       );
     });
   });
+
+  describe('when the bundle is a v0.2 bundle', () => {
```

Comment:
Should we add another test for a made up `v0.3` bundle that is the same as `v0.2`?

Reply:
👍 Added a theoretical v0.3 bundle to make sure we don't get tripped-up on an unknown media type